### PR TITLE
WIP High School Physics Rulesets

### PR DESCRIPTION
**Pull request by helenemccarron**

### CODE REVIEW

File: books/rulesets/_hs-physics-config.scss [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+/*
+  Misconception Alert note
+
+  Contains a title "Misconception Alert".
+
+  Markup: ./examples/page.note.misconception.html-cooked.html
+
+  Style guide: page.note.misconception
+*/
+
```
Comment (philschatz) [inline books/rulesets/_hs-physics-config.scss]: Can this blank line be removed? That way the comment is just above the definition

File: books/rulesets/_hs-physics-config.scss [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+$exerciseTitleContent: (
+  number: counter(exercise)
+);
+
+/*
+  Teacher Support  note
+
+  Contains a title "Teacher Support"
+
+  Markup: ./examples/page.note.teacher.html-cooked.html
```
Comment (philschatz) [inline books/rulesets/_hs-physics-config.scss]: It seems that these HTML files are missing

File: books/rulesets/_hs-physics-config.scss [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+  $pageCriticalThinking,
+  $pageProblems,
+  $pagePerformance,
+  $pageMultipleChoice,
+  $pageShortAnswer,
+  $pageExtendedResponse,
+);
+
+$bookCompositePages: (
+  //This doesn't get parsed as a list without a comma at the end
```
Comment (philschatz) [inline books/rulesets/_hs-physics-config.scss]: Could you rename `doesn't` to `does not`? That would help with reading the diff in GitHub

File: books/rulesets/_hs-physics-config.scss [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+  and have answers collated at the end of the chapter.
+
+  Markup: ./examples/book.composite.problems.html-cooked.html
+
+  Style guide: book.composite.problems
+*/
+$pageProblems: (
+  name: "Problems",
+  source: "problem",
+  //hasSolutions: true,
```
Comment (philschatz) [inline books/rulesets/_hs-physics-config.scss]: could/should this be `hasSolutions: false`?

File: books/rulesets/_hs-physics-config.scss (hunk 2 above)
Comment (helenemccarron) [inline books/rulesets/_hs-physics-config.scss]: Yes, this is a WIP, I am not done creating all files.

File: books/rulesets/_hs-physics-config.scss (hunk 4 above)
Comment (helenemccarron) [inline books/rulesets/_hs-physics-config.scss]: I will remove it when I am sure it doesn't need it, again, WIP :P

File: books/rulesets/_hs-physics-config.scss (hunk 2 above)
Comment (philschatz) [inline books/rulesets/_hs-physics-config.scss]: ah, ok. Somehow I missed that you changed the title to include "WIP" 😄

**Pull request closed without merging**